src/get-started/labels-legends-headings/index.md.njk

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout-pane.njk
+title: Making labels and legends headings
+section: Get started
+theme: How to guides
+order: 4
+description: How to markup and style labels, legends and headings
+---
+{% from "_example.njk" import example %}
+
+Whenever you design a form, you should by splitting it across multiple pages with each page containing just one thing, for example:
+
+- one piece of information you’re telling a user
+- one decision they have to make
+- one question they have to answer
+
+This makes it easier to help users to focus and understand what’s being asked of them.
+
+To help you follow this approach, you can set the contents of a `<label>` or `<legend>` of a page’s input as the page heading. Doing this means that screen reader users will only hear the contents once.
+
+For example, if you are designing a page that’s only asking users for their postcode, the question ‘What is your postcode’ can be both the label for the text input, and the page heading.
+
+{{ example({group: "get-started", item: "labels-legends-headings", example: "post-code", html: true, nunjucks: true, open: false}) }}
+
+Similarly, on a page where you are only asking users for their address, the inputs will be contained in a fieldset with legend text ‘What is your home address?’. This can also be the page heading.
+
+{{ example({group: "get-started", item: "labels-legends-headings", example: "address-group", html: true, nunjucks: true, open: false}) }}
+
+If you don’t make the legend or label the page heading, a screen reader will read out both the heading and the contents of the legend or label. If the text is the same, this will be repetitive for the user.
```

Comment:
(We talk a lot about how doing this _reduces duplication_ from the screen reader, but I think that's on the assumption that the alternative is a visually hidden label. Do we need to be more explicit about that?)